src/functions/negate.js

```diff
@@ -3,13 +3,39 @@ import {wrap} from "../core/wrap";
 // Get the complement of a function.
 export const negate = wrap({
     name: "negate",
+    summary: "Get the logical negation of a predicate function.",
+    docs: process.env.NODE_ENV !== "development" ? undefined : {
+        expects: (`
+            The function expects a single predicate function as its input.
+        `),
+        returns: (`
+            The function builds and returns a function returning true for
+            which inputs producing a falsey value when passed to the
+            inputted predicate and returning false for all inputs producing
+            a truthy value when passed to that predicate.
+        `),
+        examples: [
+            "basicUsage"
+        ],
+        related: [
+            "allPass", "anyPass", "nonePass"
+        ],
+    },
     attachSequence: false,
     async: false,
```

Comment:
> It might be enough in almost all cases to just specify some list of arguments where all of the applicable wrappers are evaluated with those arguments and their outputs are tested for equality.

That's sorta what I was thinking. At least from the beginning that should work just fine. Moving forward we'll probably want something more robust, but I'd like to first see what kind of coverage we could get using the simplest method possible.